CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -6,6 +6,9 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 ## v3.5.10 (tbd)
 
+### etcd server
+- Fix [etcdserver: fix corruption check when server has just been compacted](https://github.com/etcd-io/etcd/pull/16048)
```

Comment:
Nit - Should we remove the duplication given this is already under etcd server heading of and sentence starts with `Fix`?

```suggestion
- Fix [corruption check when server has just been compacted](https://github.com/etcd-io/etcd/pull/16048)
```